internal/build/job/manager_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/qbarrand/oot-operator/internal/build"
 	"github.com/qbarrand/oot-operator/internal/client"
 	"github.com/qbarrand/oot-operator/internal/constants"
-	"github.com/qbarrand/oot-operator/internal/registry"
+	registrypkg "github.com/qbarrand/oot-operator/internal/registry"
```

Comment:
Why not just `registry`?

Reply:
because we have a variable named registry, that is used in multiple places